packages/fxa-auth-server/lib/senders/emails/partials/appBadges/index.ts

```diff
@@ -24,13 +24,13 @@ export const appBadges = `
   </mj-section>
   <mj-section>
     <mj-column>
-      <% if (onDesktopOrTabletDevice) { %>
+      <% if (typeof onDesktopOrTabletDevice !== "undefined" && onDesktopOrTabletDevice) { %>
```

Comment:
I'm still confused about why this check is so explicit - would `<% if (onDesktopOrTabletDevice != null) { %>` not work?

Reply:
I think it's explicitly `undefined` in this template, but I'm not sure why we need to check `typeof`. Does `if (onDesktopOrTabletDevice) {` output the same?

Reply:
I'm guessing it's either going to be `undefined` or boolean, so `if (onDesktopOrTabletDevice) {` may return false negatives if the value is `false`. But if we're loosely checking `onDesktopOrTabletDevice != null` or even more strictly `onDesktopOrTabletDevice === undefined` I feel like that should work

Reply:
Checking for not null will work but we also need to check if the value is true or false

Reply:
@jodyheavener I think either way we just care if it's falsey. In our current templates, the conditional is `{{#if onDesktopOrTabletDevice}}`. So... does `if (onDesktopOrTabletDevice) {` not work? 🤔

Though... I've got latest on this branch, and in Storybook, I'm not seeing a difference in these two stories when I think the mobile version should not render the word "desktop":
<img width="1159" alt="image" src="https://user-images.githubusercontent.com/13018240/128436030-e15be8fa-0c4c-4ab4-8464-64b58bdbafde.png">

<img width="1133" alt="image" src="https://user-images.githubusercontent.com/13018240/128436042-c59a8ce6-aeae-42b9-9e53-8ddd88b4c041.png">

Reply:
`if (onDesktopOrTabletDevice) {` throws the following error so maybe we also need to check if the value is defined or not
![image](https://user-images.githubusercontent.com/37609519/128462951-8f734f66-4bba-4d5b-96b5-963725482496.png)
It's mentioned in the mustache [docs](https://www.npmjs.com/package/mustache) that `{{#variable}}` checks for undefined, null and falsy values as well.

Reply:
@LZoog thanks for bringing it to my notice, it's been fixed now!